bindings/java/src/org/sleuthkit/datamodel/ScoringManager.java

```diff
@@ -274,6 +283,49 @@ Score updateAggregateScoreAfterDeletion(long objId, Long dataSourceObjectId, Cas
 		}
 		return newScore;
 	}
+	
+	/**
+	 * Retrieves the maximum FileKnown status of any tag associated with the
+	 * object id.
+	 *
+	 * @param objectId   The object id of the item.
+	 * @param transaction The case db transaction to perform this query.
+	 *
+	 * @return The maximum FileKnown status for this object or empty.
+	 *
+	 * @throws TskCoreException
+	 */
+	private Optional<TskData.FileKnown> getTagKnownStatus(long objectId, CaseDbTransaction transaction) throws TskCoreException {
```

Comment:
I'd move this to TaggingManager and give it name of something like getMaxTagKnownStatus().

Reply:
Sounds good.